### DISCUSSION TITLE
Does ktor server support Subscriptions?

### DISCUSSION THREAD

**Discussion by lemuellew** [ExpediaGroup/graphql-kotlin#1290] [Q&A]
https://opensource.expediagroup.com/graphql-kotlin/docs/server/server-subscriptions

**Comment by smyrick** [comment]
We don't have a Ktor server library. Only a simple example. If you choose to use the generic `graphql-kotlin-server` to implement a Ktor server you can use subscriptions

**Comment by lemuellew** [comment]
Could you please give subscriptions example code for ktor  server?

**Reply by smyrick** [reply]
We don't have an example Ktor server with subscriptions. You can take a look at the spring-server which implements the Apollo `grapqh-ws` spec: https://opensource.expediagroup.com/graphql-kotlin/docs/server/spring-server/spring-subscriptions#graphql-ws-subprotocol
